ehr/resources/web/ehr/panel/BulkEditPanel.js

```diff
@@ -167,6 +167,12 @@ Ext4.define('EHR.panel.BulkEditPanel', {
         Ext4.Array.forEach(this.records, function(r){
             r.set(values);
 
+            if (this.getBoundRecord() && this.getBoundRecord().setAdditionalFields) {
```

Comment:
Let's call this `bulkPropertyAdditionalFields` or something like that. Prefixing with `set` makes it sounds like it's a function. Also, please add a comment describing the use case and an example usage for those who don't have access to the